Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:6.9.1
+
+# Packages required to run the app
+RUN npm install -g foreman
```

Comment:
@sergii-tkachenko Oh interesting, didn't know that! Let me reduce that down then